staramr/results/AMRDetectionSummary.py

```diff
@@ -67,10 +73,17 @@ def _include_detailed_negatives(self, resistance_frame: DataFrame, plasmid_frame
         negative_res_names_set = names_set - resfinder_names_set
 
         negative_entries = None
+        negative_columns = self._get_detailed_negative_columns()
 
         if len(negative_res_names_set) != len(names_set) or resistance_frame.empty:
-            negative_resistance_entries = pd.DataFrame([[x, 'None', 'Sensitive', '', ''] for x in negative_res_names_set],
-                                                       columns=('Isolate ID', 'Gene', 'Predicted Phenotype', 'Start', 'End')).set_index('Isolate ID')
+
+            if self._include_phenotype():
```

Comment:
I would shift this code into another method like `_get_negative_resistance_entries()`.

Also, it looks like the negative **Predicted Phenotypes** are showing up as `-` for me in the `summary.tsv` file:

![image](https://user-images.githubusercontent.com/200517/55987667-a750fb80-5c67-11e9-85c1-df643cb0f8d2.png)

Reply:
I've fixed that in my next commit, also the Predicted Phenotype issue is coming from a line of code where it wasn't placed properly. I moved it to the appropriate place.

Reply:
Awesome. Thank you.